tests/foreman/ui_airgun/test_dashboard.py

```diff
@@ -446,6 +451,58 @@ def test_positive_content_host_subscription_status(session):
             assert values['table'][0]['Content View'] == cv_name
 
 
+@tier2
+def test_positive_discovered_host(session):
+    """Check if the Discovered Host widget is working in the Dashboard UI
+
+    :id: 74afef58-71f4-49e1-bbb6-6d4355d385f8
+
+    :Steps:
+
+        1. Create a Discovered Host.
+        2. Navigate Monitor -> Dashboard
+        3. Review the Discovered Host Status.
+
+    :expectedresults: The widget is updated with all details.
+
+    :CaseLevel: Integration
+    """
+    org = entities.Organization().create()
+    loc = entities.Location(organization=[org]).create()
+    ipaddress = gen_ipaddr()
+    macaddress = gen_mac(multicast=False)
+    model = gen_string('alpha', length=5)
+    host_name = 'mac{0}'.format(macaddress.replace(':', ''))
+    entities.DiscoveredHost().facts(json={
+        u'facts': {
+            u'name': gen_string('alpha'),
+            u'discovery_bootip': ipaddress,
+            u'discovery_bootif': macaddress,
+            u'interfaces': 'eth0',
+            u'ipaddress': ipaddress,
+            u'macaddress': macaddress,
+            u'macaddress_eth0': macaddress,
+            u'ipaddress_eth0': ipaddress,
+            u'foreman_organization': org.name,
+            u'foreman_location': loc.name,
+            u'model': model,
+            u'memorysize_mb': 1000,
+            u'physicalprocessorcount': 2,
+        }
+    })
+
+    with session:
+        session.organization.select(org_name=org.name)
+        session.location.select(loc_name=loc.name)
+        values = session.dashboard.read('DiscoveredHosts')
+        assert len(values) == 2
```

Comment:
no-no, you have to check that there is only one value in a table, not the count of the values in your list

Reply:
@oshtaier nice catch .. It was a typo .
I have pushed the same in last commit .